deployment/template-cronjob.yaml

```diff
@@ -81,6 +81,11 @@ objects:
                   secretKeyRef:
                     key: aws_secret_access_key
                     name: openshift-probable-vulnerability-inference-result-s3
+              - name: SENTRY_DSN
+                valueFrom:
+                  secretKeyRef:
+                    name: osa
```

Comment:
@rajusem Can't we use same sentry DSN used by dependency analytics? Is it a resource created through app-interface?

Reply:
@arajkumar : We wanted to keep separate than CRDA project, for the same we have created OSA Project in sentry and using the same.